src/fs/rock/RockSwapDir.cc

```diff
@@ -84,8 +86,7 @@ Rock::SwapDir::anchorToCache(StoreEntry &entry, bool &inSync)
         return false;
 
     anchorEntry(entry, filen, *slot);
-    inSync = updateAnchoredWith(entry, *slot);
-    return true; // even if inSync is false
+    return true;
```

Comment:
Don't we need to set `entry.swap_file_sz` as the old `updateAnchoredWith()` did?

Reply:
Agreed, the existing `anchorEntry()` does(did) this already.